pmaudit

```diff
@@ -467,12 +476,8 @@ def main():
                 f.write('\n')
         sys.exit(2 if rc else 0)
 
-    db_parser = parser.add_mutually_exclusive_group()
-    db_parser.add_argument(
-        '-o', '--save', default='%s.json' % PROG,
```

Comment:
I know I keep going back and forth on this, largely because I'm gradually remembering issues that I'd forgotten from last time I worked on it, but maybe it would be better to use -j, --json here instead of -o? The reason, as noted in recent email, is that the shell has a -o option but neither dash nor bash (nor tcsh for that matter) has a -j.

Reply:
Sure, -j is fine, as long as we do it (eventually) with both pmaudit and pmash.

Reply:
Yes, that's what I am currently thinking ... use -j across the board for this option.